examples/generateCatalog.py

```diff
@@ -0,0 +1,183 @@
+                    level=logging.DEBUG,
+                    format='%(asctime)s - %(levelname)s - %(message)s')
+
+
+###########################################################
+# Utilities to be exported to python libraries
+###########################################################
+def gen_star_list(outfilename, type, fov, ra_cen, dec_cen, nstars_per_dim, seeing, filter, seed, histid):
+	sed_filename = "../sky/sed_flat.txt"
+        nstars = nstars_per_dim * nstars_per_dim
+	### Pointing from simGridStars in ImageSimulationRecipes
+
+        # add a correction for the fact that ra coordinates are smaller
+        # in angular distance by the cos(dec).
+        raCorr = math.cos(dec_cen*math.pi/180.0)
+
+        ### Coordinates uniformly spaced over the fov in a square pattern
+        if type == "random":
+	    ra = []
+            dec = []
+            dec_min = dec_cen - fov/2.0
+            dec_max = dec_cen + fov/2.0
+            ra_min = ra_cen - fov/2.0/raCorr
+            ra_max = ra_cen + fov/2.0/raCorr
+
+            for i in range(nstars):
+                ra.append(random.uniform(ra_min, ra_max))
+                dec.append(random.uniform(dec_min, dec_max))
+
+	### Coordinate ranges (degrees)
+	###  Space stars 1 arcminute apart
+        elif type == "grid":
+	    grid_radius = np.min(((nstars_per_dim / 60.) / 2., fov/2.))
+	    ra_min = ra_cen - (fov/2.0/raCorr)
+	    ra_max = ra_cen + (fov/2.0/raCorr)
+	    dec_min = dec_cen - fov/2.0
+	    dec_max = dec_cen + fov/2.0
+            ra, dec = np.meshgrid(np.linspace(ra_min, ra_max, num=nstars_per_dim),
+						  np.linspace(dec_min, dec_max, num=nstars_per_dim))
+	    ra = ra.ravel()
+	    dec = dec.ravel()
+        else:
+            raise StandardException("Illegal type requested: %s"%type)
+
+        #   print a grid distribution to show that the points are reasonably
+        #   distributed.
+        steps = 10
+        hist = np.zeros((steps, steps), dtype=int)
+        rastep = (ra_max-ra_min)/steps
+        decstep = (dec_max-dec_min)/steps
+        for i in range(len(ra)):
+            ra_bin = int((ra[i]-ra_min)/rastep)
+            dec_bin = int((dec[i]-dec_min)/decstep)
+            if ra_bin == steps and ra[i] == ra_max:
+                ra_bin = steps - 1
+            if dec_bin == steps and dec[i] == dec_max:
+                dec_bin = steps - 1
+            hist[ra_bin, dec_bin] = hist[ra_bin, dec_bin]+1
+        print "Distribution of %d stars over a %.3f field of view"%(hist.sum(), fov)
+        print hist
+
+	### Write header of instance catalog file with observing and simulation parameters
+        outfile = open(outfilename, "w")
+	print >> outfile, 'Unrefracted_RA', ra_cen
+	print >> outfile, 'Unrefracted_Dec', dec_cen
+	print >> outfile, 'Opsim_obshistid %d'%histid
+	print >> outfile, 'SIM_SEED %d'%seed
+	print >> outfile, 'SIM_EXPTIME 30.0'
+	print >> outfile, 'Unrefracted_RA 81.1760879'
```

Comment:
I mentioned this in the commit comment, but this dups the Unrefracted_RA and Dec and is hardcoded.